WalletWasabi/Blockchain/Blocks/SmartHeader.cs

```diff
@@ -35,36 +38,72 @@ public SmartHeader(uint256 blockHash, uint256 prevHash, uint height, long epochB
 
 	#region SpecialHeaders
 
-	private static SmartHeader StartingHeaderMain { get; } = new SmartHeader(
+	private static SmartHeader StartingHeaderSegwitMain { get; } = new SmartHeader(
 		new uint256("0000000000000000001c8018d9cb3b742ef25114f27563e3fc4a1902167f9893"),
 		new uint256("000000000000000000cbeff0b533f8e1189cf09dfbebf57a8ebe349362811b80"),
 		481824,
 		1503539857);
 
-	private static SmartHeader StartingHeaderTestNet { get; } = new SmartHeader(
+	private static SmartHeader StartingHeaderSegwitTestNet { get; } = new SmartHeader(
 		new uint256("00000000000f0d5edcaeba823db17f366be49a80d91d15b77747c2e017b8c20a"),
 		new uint256("0000000000211a4d54bceb763ea690a4171a734c48d36f7d8e30b51d6df6ea85"),
 		828575,
 		1463079943);
 
+	private static SmartHeader StartingHeaderTaprootMain { get; } = new SmartHeader(
```

Comment:
@thibistaken I was wondering if you could double check if the values here are set properly and I didn't make a stupid mistake?

- first taproot block hash: 0000000000000000000687bca986194dc2c1f949318629b44bb54ec0a94d8244
- previous block hash: 000000000000000000013712fc242ee6dd28476d0e9c931c75f83e6974c6bccc
- first taproot block height: 709632
- first taproot block epoch time: 1636866927

Note that epoch time can be get with Bitcoin Core RPC getblockheader command

Reply:
first taproot block hash: ✅
previous block hash: ✅
first taproot block height: ✅ 
first taproot block epoch time: ✅ I got an approximation with an epoch time converter but will double check on my node too.